selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindTextFeatureTest.java

```diff
@@ -126,7 +173,7 @@ public void setUp() throws Exception {
     ide.open(workspace);
   }
 
-  @Test
+  @Test(priority = 0)
```

Comment:
It's not a good practice to make one test dependent from another. IMHO it's better to rewrite test cases to be really independent.